### DISCUSSION TITLE
Double labeling/Overlapped labeling

### DISCUSSION THREAD

**Discussion by Amanda-Lewan** [cnr-isti-vclab/TagLab#125] [Q&A]
Hi! I was wondering if anyone could tell me if there is a way to label a traced area, and then segment parts of that traced area and assign them different labels, while keeping the original output and color coding of the entire area traced? When I try to do this separately I get variation due to human error in the tracing. Please let me know if this question is unclear and I can provide screengrab reference!

**Comment by Jordan-Pierce** [comment]
Hi @Amanda-Lewan, a screenshot might be helpful. Are you referring to a doughnut situation? Labeling the exterior as the doughnut, and the interior (the hole) as a separate entity, and having the polygons be flush against each other?
